docs/versioning.md

```diff
@@ -18,12 +23,12 @@ $ export RESTIC_PASSWORD="nlf-data-at-csc"
 
 List snapshots:
 ```
-$ restic snapshots --no-cache --no-lock
```

Comment:
Why no `--no-cache`, again (I've probably been told but don't remember)?

Reply:
Because the old `--no-cache` was accidentally copypasted in after I had debugged the slowness issue and wanted to absolutely definitely rule out poor caching performance on lustre. But it doesn't seem to be a problem on Puhti, and even if it was, the best option would be to specify cache in $TMPDIR.